Model/EventsForThirdPartyModules.php

```diff
@@ -118,19 +135,33 @@ private function prepareForListenerRun($listener)
             return [false, null];
         }
         if (isset($listener["checkClasses"])) {
-            foreach ($listener["checkClasses"] as $className) {
-                if (!$this->doesClassExist($className)) {
+            foreach ($listener["checkClasses"] as $classNameItem) {
+                $classNames = explode('|',$classNameItem);
```

Comment:
Nice idea, but can we change this is a little and support `string` and `array` as $className instead of using "|" as a delimiter?

Reply:
Well, I use string and sub-array instead